contracts/external/oracles/SaiPriceOracle.sol

```diff
@@ -0,0 +1,71 @@
+/*
+
+    Copyright 2019 dYdX Trading Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+*/
+
+pragma solidity 0.5.7;
+pragma experimental ABIEncoderV2;
+
+import { DaiPriceOracle } from "./DaiPriceOracle.sol";
+
+
+/**
+ * @title SaiPriceOracle
+ * @author dYdX
+ *
+ * PriceOracle that gives the price of Sai in USD
+ */
+contract SaiPriceOracle is DaiPriceOracle {
```

Comment:
This is confusing. It makes it seem like there are two different contracts. I think we should just use the existing `DaiPriceOracle` for both of them. It's especially confusing because this new contract is actually the one that's already deployed